pages/common/docker-containers.md

```diff
@@ -0,0 +1,36 @@
+# docker container
+
+> Manage Docker containers.
+> More information: <https://docs.docker.com/engine/reference/commandline/container/>.
+
+- List currently running Docker containers:
+
+`docker container ls`
+
+- Start one or more stopped containers:
```

Comment:
How do you specify which container to start?

Reply:
@sbrl Thanks for helping me notice there is a missing argument.